File: src/nomanssave/es.java
package nomanssave;

import java.util.Comparator;

class es implements Comparator {
   public int a(er var1, er var2) {
      return var1.name.compareTo(var2.name);
   }

   // $FF: synthetic method
   public int compare(Object var1, Object var2) {
      return this.a((er)var1, (er)var2);
   }
}

File: NMSSaveEditor/nomanssave/lower/es.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace NMSSaveEditor
{

class es : IComparer {
   public int a(er var1, er var2) {
      return var1.name.CompareTo(var2.name);
   }
   public int compare(object var1, object var2) {
      return this.a((er)var1, (er)var2);
   }
}

}
